documentdb/spring-data-azure-documentdb/src/test/java/com/microsoft/azure/spring/data/documentdb/repository/config/AbstractDocumentDbConfiguratinUnitTest.java

```diff
@@ -27,10 +27,10 @@
 
     @Test
     public void containsDocumentDbFactory() throws ClassNotFoundException {
-        final AbstractApplicationContext context = new AnnotationConfigApplicationContext(
-                TestDocumentDbConfiguration.class);
-
-        assertThat(context.getBean(DocumentDbFactory.class)).isNotNull();
+        try (AbstractApplicationContext context = new AnnotationConfigApplicationContext(
```

Comment:
I am using Spring Tool Suite IDE along with SonarLint then it is showing that context is not closed and we will have resource leak to fix that using try here